src/quo2/components/record_audio/record_audio/view.cljs

```diff
@@ -0,0 +1,473 @@
+(ns quo2.components.record-audio.record-audio.view
+  (:require [quo.react :refer [memo effect!]]
+            [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icons]
+            [quo2.components.buttons.button :as button]
+            [react-native.reanimated :as reanimated]
+            [reagent.core :as reagent]
+            [cljs-bean.core :as bean]
+            [quo2.components.record-audio.record-audio.style :as style]
+            [status-im.utils.utils :as utils]))
+
+(defonce recording? (reagent/atom false))
+(defonce locked? (reagent/atom false))
+(defonce ready-to-send? (reagent/atom false))
+(defonce ready-to-lock? (reagent/atom false))
+(defonce ready-to-delete? (reagent/atom false))
+(defonce reviewing-audio? (reagent/atom false))
+(defonce clear-timeout (atom nil))
+(defonce record-button-at-initial-position? (atom true))
+(defonce record-button-is-animating? (atom false))
+(defonce touch-active? (atom false))
+(defonce recording-timer (atom nil))
+(defonce recording-length-ms (atom 0))
+
+(def ^:private scale-to-each 1.8)
+(def ^:private scale-to-total 2.6)
+(def ^:private scale-padding 0.16)
+(def ^:private opacity-from-lock 1)
+(def ^:private opacity-from-default 0.5)
+(def ^:private signal-anim-duration 3900)
+(def ^:private signal-anim-duration-2 1950)
+
+(def ^:private animated-ring
+  (reagent/adapt-react-class
+   (memo
+    (fn [props]
+      (let [{:keys [scale opacity color]} (bean/bean props)]
```

Comment:
do we need this bean library here? can it be done without it? Are we still intending to use the library? 🤔